failed to get artifact 'v0.24.0': loading image from docker engine: Error response from daemon: client version 1.22 is too old. Minimum supported API version is 1.24, please upgrade your client to a newer version

Running `make run` I get this error:

```
DEMO SETUP => create weave gitops component
processing componentfile.yaml...
  processing document 1...
    processing index 1
found 1 component
adding component ocm.software/demos/weave-gitops:v1.0.0...
  adding resource ociImage: "name"="image","version"="<componentversion>"...
    image ghcr.io/weaveworks/wego-app:v0.24.0
Error: failed adding component "ocm.software/demos/weave-gitops"(componentfile.yaml[1][1]): cannot get resource blob for "image"(componentfile.yaml[1][1][resources][0]): failed to get artifact 'v0.24.0': loading image from docker engine: Error response from daemon: client version 1.22 is too old. Minimum supported API version is 1.24, please upgrade your client to a newer version
make[1]: *** [build] Error 1
make: *** [run] Error 2

```

--- a/00-setup-demo/charts/nginx-ingress/Chart.yaml
+++ b/00-setup-demo/charts/nginx-ingress/Chart.yaml
@@ -6,7 +6,7 @@ icon: https://raw.githubusercontent.com/nginxinc/kubernetes-ingress/v3.1.1/deplo
 keywords:
 - ingress
 - nginx
-kubeVersion: '>= 1.22.0-0'
+kubeVersion: '>= 1.28.0-0'
 maintainers:
 - email: kubernetes@nginx.com
   name: nginxinc

--- a/00-setup-demo/charts/nginx-ingress/templates/controller-service.yaml
+++ b/00-setup-demo/charts/nginx-ingress/templates/controller-service.yaml
@@ -20,7 +20,7 @@ spec:
   {{- end }}
 {{- end }}
 {{- if eq .Values.controller.service.type "LoadBalancer" }}
-  {{- if and (semverCompare ">=1.22.0-0" .Capabilities.KubeVersion.Version) (.Values.controller.service.allocateLoadBalancerNodePorts) }}
+  {{- if and (semverCompare ">=1.28.0-0" .Capabilities.KubeVersion.Version) (.Values.controller.service.allocateLoadBalancerNodePorts) }}
   allocateLoadBalancerNodePorts: {{ .Values.controller.service.allocateLoadBalancerNodePorts }}
   {{- end }}
   {{- if .Values.controller.service.loadBalancerIP }}

--- a/00-setup-demo/lib.sh
+++ b/00-setup-demo/lib.sh
@@ -156,8 +156,8 @@ function configure-gitea {
             --docker-username=ocm-admin \
             --docker-password=$TOKEN
 
-    docker tag ghcr.io/phoban01/podinfo:6.3.5-static gitea.ocm.dev/software-provider/podinfo:6.3.5-static
-    docker tag ghcr.io/phoban01/podinfo:6.3.6-static gitea.ocm.dev/software-provider/podinfo:6.3.6-static
+    docker tag ghcr.io/open-component-model/podinfo:6.3.5-static gitea.ocm.dev/software-provider/podinfo:6.3.5-static
+    docker tag ghcr.io/open-component-model/podinfo:6.3.6-static gitea.ocm.dev/software-provider/podinfo:6.3.6-static
     docker push gitea.ocm.dev/software-provider/podinfo:6.3.5-static
     docker push gitea.ocm.dev/software-provider/podinfo:6.3.6-static
 }

--- a/00-setup-demo/lib_vars.sh
+++ b/00-setup-demo/lib_vars.sh
@@ -16,8 +16,8 @@ os=$(uname -s)
 tools=(helm flux kind jq kubectl ocm mkcert tea git curl docker gzip)
 
 images=(
-ghcr.io/phoban01/podinfo:6.3.5-static
-ghcr.io/phoban01/podinfo:6.3.6-static
+ghcr.io/open-component-model/podinfo:6.3.5-static
+ghcr.io/open-component-model/podinfo:6.3.6-static
 ghcr.io/weaveworks/wego-app:v0.24.0
 ghcr.io/fluxcd/helm-controller:v0.33.0
 ghcr.io/fluxcd/kustomize-controller:v1.0.0-rc.3
@@ -34,9 +34,8 @@ registry.k8s.io/kube-controller-manager:v1.27.1
 registry.k8s.io/kube-proxy:v1.27.1
 registry.k8s.io/kube-scheduler:v1.27.1
 docker.io/kindest/local-path-provisioner:v20230511-dc714da8
-ghcr.io/open-component-model/ocm-controller:v0.8.1
-ghcr.io/open-component-model/ocm-registry:v0.0.1-dev
-ghcr.io/phoban01/replication-controller:v0.2.4
+ghcr.io/open-component-model/ocm-controller:v0.19.0
+ghcr.io/open-component-model/replication-controller:v0.13.0
 registry:2
 cgr.dev/chainguard/busybox
 docker.io/library/alpine:latest
@@ -50,12 +49,12 @@ gcr.io/tekton-releases/github.com/tektoncd/triggers/cmd/controller:v0.24.0
 gcr.io/tekton-releases/github.com/tektoncd/triggers/cmd/eventlistenersink:v0.24.0
 gcr.io/tekton-releases/github.com/tektoncd/triggers/cmd/interceptors:v0.24.0
 gcr.io/tekton-releases/github.com/tektoncd/triggers/cmd/webhook:v0.24.0
-ghcr.io/phoban01/ocm:0.3.0-rc.2
+ghcr.io/open-component-model/ocm:latest
 )
 
 preloadimages=(
-ghcr.io/phoban01/podinfo:6.3.5-static
-ghcr.io/phoban01/podinfo:6.3.6-static
+ghcr.io/open-component-model/podinfo:6.3.5-static
+ghcr.io/open-component-model/podinfo:6.3.6-static
 ghcr.io/weaveworks/wego-app:v0.24.0
 ghcr.io/fluxcd/helm-controller:v0.33.0
 ghcr.io/fluxcd/kustomize-controller:v1.0.0-rc.3
@@ -66,9 +65,8 @@ registry.k8s.io/ingress-nginx/kube-webhook-certgen:v20230312-helm-chart-4.5.2-28
 registry.k8s.io/ingress-nginx/controller:v1.7.1
 registry.k8s.io/coredns/coredns:v1.10.1
 docker.io/kindest/local-path-provisioner:v20230511-dc714da8
-ghcr.io/open-component-model/ocm-registry:v0.0.1-dev
-ghcr.io/open-component-model/ocm-controller:v0.8.1
-ghcr.io/phoban01/replication-controller:v0.2.4
+ghcr.io/open-component-model/ocm-controller:v0.19.0
+ghcr.io/open-component-model/replication-controller:v0.13.0
 registry:2
 cgr.dev/chainguard/busybox
 docker.io/library/alpine:latest
@@ -82,7 +80,7 @@ gcr.io/tekton-releases/github.com/tektoncd/triggers/cmd/controller:v0.24.0
 gcr.io/tekton-releases/github.com/tektoncd/triggers/cmd/eventlistenersink:v0.24.0
 gcr.io/tekton-releases/github.com/tektoncd/triggers/cmd/interceptors:v0.24.0
 gcr.io/tekton-releases/github.com/tektoncd/triggers/cmd/webhook:v0.24.0
-ghcr.io/phoban01/ocm:0.3.0-rc.2
+ghcr.io/open-component-model/ocm:latest
 )
 
 helm_mac_instructions="brew install helm"

--- a/00-setup-demo/manifests/replication.yaml
+++ b/00-setup-demo/manifests/replication.yaml
@@ -379,7 +379,7 @@ spec:
         - --zap-log-level=4
         command:
         - /manager
-        image: ghcr.io/phoban01/replication-controller:v0.2.4
+        image: ghcr.io/open-component-model/replication-controller:v0.13.0
         livenessProbe:
           httpGet:
             path: /healthz

--- a/00-setup-demo/tekton/publish_component_pipeline.yaml
+++ b/00-setup-demo/tekton/publish_component_pipeline.yaml
@@ -88,7 +88,7 @@ spec:
         path: ocm-config.yaml
   steps:
   - name: build
-    image: ghcr.io/phoban01/ocm:0.3.0-rc.2
+    image: ghcr.io/open-component-model/ocm:latest
     workingDir: "$(workspaces.source.path)"
     args:
       - add
@@ -100,7 +100,7 @@ spec:
       - v3alpha1
       - componentfile.yaml
   - name: sign
-    image: ghcr.io/phoban01/ocm:0.3.0-rc.2
+    image: ghcr.io/open-component-model/ocm:latest
     workingDir: "$(workspaces.source.path)"
     args:
       - sign
@@ -118,7 +118,7 @@ spec:
     - name: certs
       mountPath: /etc/ssl/certs
   - name: transfer
-    image: ghcr.io/phoban01/ocm:0.3.0-rc.2
+    image: ghcr.io/open-component-model/ocm:latest
     workingDir: "$(workspaces.source.path)"
     args:
       - --config
@@ -134,7 +134,7 @@ spec:
     - name: ocm-config
       mountPath: /var/ocm
   - name: verify
-    image: ghcr.io/phoban01/ocm:0.3.0-rc.2
+    image: ghcr.io/open-component-model/ocm:latest
     workingDir: "$(workspaces.source.path)"
     args:
       - --config
@@ -156,7 +156,7 @@ spec:
     - name: ocm-config
       mountPath: /var/ocm
   - name: view
-    image: ghcr.io/phoban01/ocm:0.3.0-rc.2
+    image: ghcr.io/open-component-model/ocm:latest
     workingDir: "$(workspaces.source.path)"
     args:
       - --config

